recipe/cache.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Utility\Httpie;
+
+desc('Reset the opcache using a file strategy');
+task('sumo:opcache:reset-file', function () {
+    $opcacheResetScript = 'opcache_reset.php';
+    $scriptPath = '{{ release_path }}/public/' . $opcacheResetScript;
+
+    run('echo "<?php opcache_reset();" > ' . $scriptPath);
+
+    $response = Httpie::get(get('production_url') . '/' . $opcacheResetScript)->send();
+    if ($response === false) {
+        writeln('Could not perform an opcache reset via file.');
```

Comment:
I would enclose the message in `<comment>`, as this will output it in yellow, so it is more visible. See https://github.com/sumocoders/deployer-sumo/blob/master/recipe/symlink.php#L32 for an example.

Reply:
Fixed